shared/datastore_medium_test.go

```diff
@@ -1,6 +1,6 @@
 // +build medium
 
-package shared
+package shared_test
```

Comment:
Nit: `sharedtest` to match the name of the directory.

Reply:
This is actually a deliberate convention. `foo/*_test.go` files may be `package foo` or `package foo_test`. This is `shared_test` to a void a dependency cycle between `shared` and `shared_test`.

Reply:
Hah I see, so it's also different from `shared/sharedtest`.